aepsych/plotting.py

```diff
@@ -154,9 +154,10 @@ def _plot_strat_1d(
     x, y = strat.x, strat.y
     assert x is not None and y is not None, "No data to plot!"
 
-    grid = strat.model.dim_grid(gridsize=gridsize)
-    samps = norm.cdf(strat.model.sample(grid, num_samples=10000).detach())
-    phimean = samps.mean(0)
+    if strat.model is not None:
+        grid = strat.model.dim_grid(gridsize=gridsize)
+        samps = norm.cdf(strat.model.sample(grid, num_samples=10000).detach())
+        phimean = samps.mean(0)
 
```

Comment:
It's weird here that `if strat.model is None`, the rest of the code breaks. I know that this for the purposes mypy errors, so ultimately it's unlikely to matter. But the code should still make sense locally. Will this be fixed if strat.model is None, it raises an exception?

Reply:
Okay, I will add a condition to raise an exception.